### PR TITLE
Two more UI finesse fixes

### PR DESCRIPTION
**Pull request by cbjeukendrup**
Resolves: #9869 
Resolves: #10045 

- Fix updating color of arrow in StyledPopupView
- Fix height of NoteInputBar (again)

### CODE REVIEW

File: src/appshell/qml/NotationPage/NotationPage.qml [hunk 1]
```diff
@@ -182,6 +182,8 @@ DockPage {
                 root.toolBarRightDropDestination
             ]
 
+            thickness: orientation === Qt.Horizontal ? 40 : 76
```
Comment (RomanPudashkin) [inline src/appshell/qml/NotationPage/NotationPage.qml:185]: Could you please explain what is the purpose of this change?

Comment (cbjeukendrup) [reply src/appshell/qml/NotationPage/NotationPage.qml:185]: By design, the Note Input Bar should be 40px high when horizontal, and 72 wide when vertical. 
The other toolbar should still be 36 high.

Comment (RomanPudashkin) [reply src/appshell/qml/NotationPage/NotationPage.qml:185]: We already have these size limits. The bar should use them
Screenshot 2021-11-25 at 14 16 19

Comment (cbjeukendrup) [reply src/appshell/qml/NotationPage/NotationPage.qml:185]: It is a bit inconsistent. In the other (non-floatable) toolbars, the content component depends on the fact that the height (thickness) of the toolbar is hardcoded to 36. Instead of hardcoding this value, I made a changeable property, because it is different for the NoteInputBar. 

For floating toolbars, the content height of the toolbars is used. This is mainly for the PlaybackToolbar, which should have a different height when floating, not the default 36px. Because of this, we also have to specify the desired height and width of the NoteInputBar, so that it gets the correct width when it's floating too.

Eventually we could make this consistent: 
- either make sure that for each toolbar the content item always specifies the correct size, and then use the size of that content item to determine the minimum and maximum size of the toolbar
- or use the `thickness` property consistently, so also for the PlaybackToolbar when it's floating.

The latter is doable but is not really nicer than what we have now, and the former is more work and has the disadvantage that we manually need to make sure that all four top toolbars have the same implicit content height of 36.

So what would you consider the best option?

Comment (cbjeukendrup) [reply src/appshell/qml/NotationPage/NotationPage.qml:185]: I tried various solutions, including the ones I mentioned in the previous comment, but also, using the maximum of the content height and the default thickness (36). See https://github.com/cbjeukendrup/MuseScore/commit/4e6cb60830f8fe9b7dd7891acd3070e785fd0890. But with everything I tried, it goes wrong when re-docking the Playback Toolbar:

https://user-images.githubusercontent.com/48658420/145690765-af7c9c0b-6184-4a0a-b79a-de78cfa3e275.mov

With the current situation in this PR, that problem does not occur. So maybe it's a good option to merge this as it is now, so that at least the visual with the note input bar, and then we can investigate later why other solutions don't work well with the Playback Toolbar.

**Comment by jessjwilliamson** [comment]
Tested this on Windows, looks fixed to me! :)

**Review by RomanPudashkin — APPROVED** [review]
